app/models/crime_application.rb

```diff
@@ -123,4 +123,10 @@ def applicant_18_or_over_at_date_stamp?
     age >= 18
   end
   # rubocop:enable Metrics/AbcSize
+
+  def benefit_check_recipient
+    return partner if partner.present? && partner.has_passporting_benefit?
+
+    applicant
```

Comment:
The logic for this might need refactoring, I think it will be clearer when working on the cya page and tasklist

Reply:
Could this live in TypeOfMeansAssessment for no? (Although I do think we should also explore a separate object to model the benefit check which would understand who the check was being run against, where there was forthcoming information etc.)